core/src/com/unciv/ui/multiplayer/ViewFriendsListScreen.kt

```diff
@@ -0,0 +1,92 @@
+package com.unciv.ui.multiplayer
+
+import com.badlogic.gdx.scenes.scene2d.ui.*
+import com.unciv.ui.pickerscreens.PickerScreen
+import com.unciv.ui.popup.Popup
+import com.unciv.ui.utils.*
+import com.unciv.ui.utils.AutoScrollPane as ScrollPane
+
+class ViewFriendsListScreen(previousScreen: BaseScreen) : PickerScreen() {
+    private val rightSideTable = Table()
+    private val leftSideTable = Table()
+    private var friendsTable = Table()
+
+    private val addFriendButton = "Add friend".toTextButton()
+    private val editFriendButton = "Edit friend".toTextButton()
+
+    private val friendsList = FriendList().getFriendsList()
+    private var listOfFriendsButtons = arrayListOf<TextButton>()
+
+    private lateinit var selectedFriend: FriendList.Friend
+
+    init {
+        setDefaultCloseAction(previousScreen)
+
+        //Help Button Setup
+        val tab = Table()
+        val helpButton = "Help".toTextButton()
+        helpButton.onClick {
+            val helpPopup = Popup(this)
+            helpPopup.addGoodSizedLabel("To add a friend, ask him to send you his player ID.").row()
+            helpPopup.addGoodSizedLabel("Click the 'Add friend' button.").row()
+            helpPopup.addGoodSizedLabel("Insert his player ID and a name for him.").row()
+            helpPopup.addGoodSizedLabel("Then click the 'Add friend' button again.").row()
+            helpPopup.addGoodSizedLabel("").row()
+
+            helpPopup.addGoodSizedLabel("After that you will see him in your friends list.").row()
+            helpPopup.addGoodSizedLabel("").row()
```

Comment:
I don't think an empty label has a different height than an empty Cell - even then a pad would help. So just thingy.row() - or .add().row() if you want to manipulate the Cell before the row() call which changes the chain. Like .add().pad(10f).row() pads the empty cell but .row().pad() finishes the row then sets a default padding for the next row.

But then I would probably have stored all the lines in a raw string and used just one Label... Look for `""".trimIndent()` in the project if you're curious. Not that I _like_ trimIndent..